test/JDBC/input/collation_tests-vu-prepare.sql

```diff
@@ -0,0 +1,116 @@
+--varchar
+CREATE TABLE english_collation_varchar (name varchar(20) COLLATE latin1_general_ci_as);
```

Comment:
Object names should be prefixed with test file name.

Reply:
Updated in the latest commit.